docs/guides/building-forms.md

```diff
@@ -235,7 +214,7 @@ If you submit this form, it will submit the data to the forms API endpoint `/api
 
 ## Part 4: Configuring Forms in Next.js
 
-You have created a back-end server for form submission. Now it's time to configure the client (the form itself) inside Next.js using React. The first step will be extending your knowledge of HTML forms and converting it into React (using [JSX](https://reactjs.org/docs/introducing-jsx.html)).
+You have created a Nextjs API route for form submission. Now it's time to configure the client (the form itself) inside Next.js using React. The first step will be extending your knowledge of HTML forms and converting it into React (using [JSX](https://reactjs.org/docs/introducing-jsx.html)).
```

Comment:
Sorry to be picky here, but we are mixing up Nextjs and Next.js. Lets go with Next.js

Reply:
Done done. ✅